components/gitpod-cli/BUILD.yaml

```diff
@@ -9,7 +9,20 @@ packages:
       - CGO_ENABLED=0
       - GOOS=linux
     deps:
+      - :version
       - components/supervisor-api/go:lib
       - components/gitpod-protocol/go:lib
+    argdeps:
+      - version
     config:
       packaging: app
+    prep:
+      - ["cp", "_deps/components-gitpod-cli--version/version.txt", "pkg/gitpod/version.txt"]
```

Comment:
It would be cool to have `gp version` command to verify it.

Reply:
Sounds like a great idea

Reply:
This should be implemented now